nrf_modem/doc/dectphy.rst

```diff
@@ -0,0 +1,296 @@
+
+When the operation has completed, its result is signalled to the application in the :c:member:`nrf_modem_dect_phy_callbacks.op_complete` callback function.
+If any error has occurred in scheduling or executing the operation, it is returned in the callback.
+
+Reception modes
+---------------
+
+A reception operation can have three different reception modes:
+
+* Continuous: the reception is continued after PDC reception, up to its ``duration``.
+* Semi-continuous: the reception is continued after PDC reception, unless it's a unicast PDC.
+* One-shot: the reception is terminated after PDC reception.
+
+In all reception modes, the reception will not extend past the operation duration specified in the :c:member:`nrf_modem_dect_phy_rx_params.duration` parameter.
+
+Reception with RSSI measurement
+-------------------------------
+
+A reception operation may be combined with an RSSI measurement operation by configuring the :c:member:`nrf_modem_dect_phy_rx_params.rssi_interval` parameter.
+
+Stopping reception
+------------------
+
+It is possible to stop the execution of reception operations by calling the :c:func:`nrf_modem_dect_phy_rx_stop` function.
+
+A reception operation may only be stopped when it is currently being executed.
+
+It is not possible to unschedule the execution of reception operations (or any others).
+
+RSSI measurement
+================
+
+The application can schedule an RSSI measurement by operation by calling the :c:func:`nrf_modem_dect_phy_rssi` function.
+
+The operation is asynchronous, and the completion of the :c:func:`nrf_modem_dect_phy_rssi` function only signals that the request was sent to the modem.
+
+RSSI measurements are sent to the application in the :c:member:`nrf_modem_dect_phy_callbacks.rssi` callback function at a configurable interval,
+as specified by the :c:member:`nrf_modem_dect_phy_rssi_params.reporting_interval` field in the operation parameters.
+
+When the operation has completed, its result is signalled to the application in the :c:member:`nrf_modem_dect_phy_callbacks.op_complete` callback function.
+If any error has occurred in scheduling or executing the operation, it is returned in the callback.
+
+Combined TX/RX operations
+=========================
+
+It is possible to schedule a combined TX and RX operation by calling the :c:func:`nrf_modem_dect_phy_tx_rx` function.
+Scheduling two operations at once has the advantage of being faster, that is, has a lower latency than scheduling the two halves of the operation separately.
+
+The operations will be executed one after the other, starting with the TX operation.
+The RX operation will be executed only if the TX operation has completed successfully.
+
+The start time of the RX operation is relative to the completion of the TX operation.
+
```

Comment:
Maybe it could be clarified here both TX and Rx part are completed with own callbacks and that the order of callbacks is always the same : first TX and then the RX part.